### DISCUSSION TITLE
Version 2.2 on Buster (RPi 3): Which OS and Phoniebox version to upgrade to?

### DISCUSSION THREAD

**Discussion by retrofreak83** [MiczFlor/RPi-Jukebox-RFID#2067] [Q&A] [answered]
Dear people of Phoniebox,

my installation is running on a system as described in the discussion title:
- Raspberry Pi 3
- Raspbian Buster
- Phoniebox version 2.2

The Debian system is outdated, but the Phoniebox installation is also, I wonder what to do.
The documentation for Phoniebox 2.x speaks only about using Buster vs. Bullseye, but says that some people have "issues" when using Bullseye. I'm not sure what these "issues" are and also whether that information is still valid.
But since the recent Debian stable is Bookworm, I would like to upgrade to that instead.

Apart from that, there is Phoniebox 3 but I'm unsure whether this would be the recommendation to upgrade to or if it is not stable enough right now.

Thanks in advance

**Comment by retrofreak83** [comment]
Absolutely no reaction? 😢

**Comment by AlvinSchiller** [comment]
Hi,
i think the most issues with bullseye are resolved, but can't tell for sure. It should most likely work.
I haven' read about any experience with Bookworm, but would be nice to hear if you give it a try.

If you just want to update then sticking to Version 2 would be necessery. Version 3 has no update path from Version 2, so a new installation is needed. Version 3 still lacks some features though.

So it will highly depend on your usecase whats best for you and if you are willing to try out the options.

**Comment by s-martin** [comment, marked as answer]
bullseye should work now, for bookworm I'm not aware, if anybody has tried that yet, but tests are always welcome.

2.x is stable (I suggest using the develop branch). Spotify would only work for 2.x

3.x is pretty mature already, but may have some rough edges.
Please check, if it has all the features you want here: https://rpi-jukebox-rfid.readthedocs.io/en/latest/featurelist.html

**Comment by retrofreak83** [comment]
Thank you both vor your answers,
for now, I am just looking for something that gives me a more recent and maintained Debian version. And I guess in bullseye it should be easy/easier to get da working youtube-dl.
